examples/sample_measurement/pyproject.toml

```diff
@@ -12,9 +12,16 @@ click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pal
 [tool.poetry.group.dev.dependencies]
 mypy = ">=1.0"
 grpc-stubs = "^1.53"
+types-protobuf = "^4.21"
 # Uncomment to use prerelease dependencies.
-# ni-measurementlink-service = {path = "../..", develop = true}
+ni-measurementlink-service = {path = "../..", develop = true}
```

Comment:
This should be commented out when you check it in.

Reply:
Fixed as suggested.

Reply:
I previously asked Wesley to uncomment this so the example will use his update code. Otherwise, it uses ni-measurementlink-service 1.1.0, which doesn't support protobuf enums.

Reply:
Sorry

Reply:
Uncommented again so that the checked in code uses prerelease dependencies. I wasn't entirely sure if this was something we did locally or if we actually wanted this change in the repo.

Reply:
I want this change in the repo until we build a prerelease that has protobuf enum support, then we can comment it back out.

We need to figure out a better workflow for this. :(